scripts/retdec.py

```diff
@@ -48,8 +48,8 @@ def do_invoke(self, argv):
             "raw_endian": "big" if is_big_endian() else "little",
         }
 
-        raw_cmd = "{}  -m {} --raw-section-vma {} --raw-entry-point {} -e {} -f plain -a {} -o {} -l {} {} --cleanup"
-        bin_cmd = "{}  -m {} -e {} -f plain -a {} -o {} -l {} {} --cleanup"
+        raw_cmd = "'{}' -m {} --raw-section-vma {} --raw-entry-point {} -e {} -f plain -a {} -o {} -l {} '{}' --cleanup"
```

Comment:
As @Grazfather suggested, it's better to use `shlex.quote` instead (see https://docs.python.org/3/library/shlex.html#shlex.quote) because with this patch, if the path name contains a `'` (quote) we'd be back to the same point.
This should be a really minor change.